assets/js/admwpp.js

```diff
@@ -9,5 +9,32 @@
          $('.admwpp-custom-select').selectric();
       }
 
+      var dateFormat = "mm/dd/yy",
```

Comment:
Use the global date format used in globals.php passing it to the JS with localize_scripts.
If there is none in the globals.php define one such: `define('ADMWPP_SEARCH_DATE_FORMAT', 'mm/dd/yy');`

Reply:
Done